### PR TITLE
Improve Intro to python notebook

### PR DESCRIPTION
**Pull request by aguspesce**
- Improve the code and text
- Add more formative assessments
- Improve homework

### CODE REVIEW

**Review by mdpiper — APPROVED** [review]
Thank you for all your work on this, @aguspesce!

**Comment by mdpiper** [comment]
@aguspesce Would you like to merge this PR, or would you prefer I do so?

**Comment by aguspesce** [comment]
@mdpiper. I am waiting to @isamarcortes review

File: coco_2_python_basics.ipynb [hunk 1]
```diff
@@ -3,45 +3,57 @@
   {
```
Comment (aguspesce) [inline coco_2_python_basics.ipynb:3]: <p>Before doing your homework, <em>check</em> that you have a _remote  repository_ in your GitHub account called <code>coco_coding_homework</code>. </p><p>If you don't have it, _create this remote repository_ according to what you learned in the  <em>_git </em>lesson_. </p><p>In this repository you will push all the notebooks with the solution of the proposed activities.</p><p> </p><p>Then, create a notebook called <code>homework_week_3.ipynb</code> into the _local repository_ related with the remote repository. </p><p>There you will solve the activities proposed below and <em>push </em>it to the remote repository in GitHub. </p><p>In this way, one of the trainers will be able to see your answers and give you feedback. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='127'/>

**Pull request merged**